### DISCUSSION TITLE
[Loki] - Keep applied format when using "show context"

### DISCUSSION THREAD

**Discussion by nicovak** [grafana/grafana#53473] [Observability]
**Is your feature request related to a problem? Please describe.**
I would like to keep the applied format to logs when using "**show context**" for a specific line. For example, If we use JSON format + Line format, It's not applied to "**show context**" results.

**Describe the solution you'd like**
Any specified formats are applied to "**show context**" results

**Describe alternatives you've considered**
None

**Additional context**

- Example LogQL

```
{app="example"} |= "ERROR" | json | line_format "{{.message}}"
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
